tmpl/blog/mirage-seal.md

```diff
@@ -0,0 +1,85 @@
+# Unikernels: HTTP -> HTTPS
+
+Building a static website is one of the better-supported user stories for MirageOS, but it currently results in an HTTP-only site, with no capability for TLS.  Although there's been a great TLS stack [available for a while now](http://openmirage.org/blog/introducing-ocaml-tls), it was a bit fiddly to assemble the pieces of TLS, Cohttp, and the MirageOS frontend tool in order to construct an HTTPS unikernel.  With MirageOS 2.5, that's changed!  Let's celebrate by building an HTTPS-serving unikernel of our very own.
+
+## Prerequisites
+
+## Get a Certificate
+
+To serve HTTPS, we'll need a certificate to present to clients (i.e., browsers) for authentication and establishing asymmetric encryption. For just testing things out, or when it's okay to cause a big scary warning message to appear for anyone browsing a site, we can just use a self-signed certificate.  Alternatively, the domain name registrar or hosting provider for a site will be happy to sell (or in some cases, give!) a certificate -- both options are explained in more detail below.
+
+Whichever option you choose, you'll need to install `ocaml-certify` to get started (assuming you'd like to avoid using `openssl`).  To do so, pin the package in opam:
+
+```
+opam pin add ocaml-certify https://github.com/yomimono/ocaml-certify.git
```

Comment:
Any reason why the package name has an "ocaml-" prefix?  Convention would be to name this:

```
opam pin add certify https://github.com/yomimono/ocaml-certify.git
```